pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1614,242 +1614,10 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 	newGlobalStatus.ConfigItems = make(map[string]types.ConfigItemStatus)
```

Comment:
You shold have a routine in global.go - NewGlobalStatus() that returns a global status with initialized maps..

newGlobalStatus := types.NewGlobalStatus()